lib/nationbuilder/client.rb

```diff
@@ -68,6 +68,26 @@ def response
     Thread.current[:nationbuilder_rb_response]
   end
 
+  def next?
+    @next ? true : false
+  end
+
+  def next
```

Comment:
We should not add these methods to the client. I think a proper implementation of pagination would involve an external pagination class. The interface would be something similar to this:

```ruby
result = client.call(:basic_pages, :index, site_slug: 'organizeralexandreschmitt')
paginated = Paginator.new(result)
page1 = paginated
page2 = page1.next_page
page3 = page2.next_page
...
```